src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/LocalAccess.cs

```diff
@@ -417,14 +419,16 @@ internal MutableBox(int index, Type type)
             }
 
             [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2077:UnrecognizedReflectionPattern",
-                Justification = "_type is a ValueType. You can always create an instance of a ValueType.")]
+                Justification = "_type is a ValueType. You can always get an uninitialized ValueType.")]
             public override int Run(InterpretedFrame frame)
             {
                 object? value;
 
                 try
                 {
-                    value = Activator.CreateInstance(_type);
+                    value = _type.IsNullableType() ?
```

Comment:
It wasn't clear to me why we want to use Activator.CreateInstance when the type is a `System.Nullable<T>`. Does something different/unexpected happen if we use `RuntimeHelpers.GetUninitializedObject(_type)` here unconditionally?

Reply:
> Does something different/unexpected happen if we use `RuntimeHelpers.GetUninitializedObject(_type)` here unconditionally?

Yes, `GetUninitializedObject(_type)` will return `(ValueType?)default(ValueType)` - that is, a boxed instance of `ValueType` rather than `null` when `_type == typeof(ValueType?)`.